test/functionalTest/cases/1960_failure_stats_in_db_and_notifications/failure_stats_in_db_and_notifications.test

```diff
@@ -0,0 +1,510 @@
+# Copyright 2016 Telefonica Investigacion y Desarrollo, S.A.U
```

Comment:
Have you tested this .test in noCache mode? (jenkins will do for us, but it would be interesting to know in advance)